README.adoc

```diff
@@ -263,58 +251,56 @@ include::finish/resources/CustomConfigSource.json[tags=**]
+----
+#Create the `HealthIT` class.#
+`src/test/java/it/io/openliberty/guides/health/HealthIT.java`
+----
 
-HealthTest.java
-[source, java, linenums, role='code_column']
+HealthIT.java
+[source, java, linenums, role='code_column hide_tags=copyright tags=HealthIT']
 ----
-include::finish/src/test/java/it/io/openliberty/guides/health/HealthTest.java[tags=HealthTest]
+include::finish/src/test/java/it/io/openliberty/guides/health/HealthIT.java[]
 ----
 
 Let's break down the test cases:
 
-- The [hotspot=25-33 file=0]`testIfServicesAreUp()` test case compares the generated health report 
-with the actual status of the services. 
-- The [hotspot=36-42 file=0]`testReadiness()` test case compares the generated health report for the 
-readiness checks with the actual status of the services. 
-- The [hotspot=45-51 file=0]`testLiveness()` test case compares the generated health report for the 
-liveness checks with the actual status of the services. 
-- The [hotspot=54-68 file=0]`testIfInventoryServiceIsDown()` test case puts the `inventory` service 
-in maintenance by setting the [hotspot=3 file=1]`io_openliberty_guides_inventory_inMaintenance` 
-property to `true` and comparing the generated health report with the actual status of 
+- The [hotspot=testIfServicesAreUp file=0]`testIfServicesAreUp()` test case compares the generated health report
+with the actual status of the services.
+- The [hotspot=testReadiness file=0]`testReadiness()` test case compares the generated health report for the
+readiness checks with the actual status of the services.
+- The [hotspot=testLiveness file=0]`testLiveness()` test case compares the generated health report for the
+liveness checks with the actual status of the services.
+- The [hotspot=testIfInventoryServiceIsDown file=0]`testIfInventoryServiceIsDown()` test case puts the `inventory` service
+in maintenance by setting the [hotspot=3 file=1]`io_openliberty_guides_inventory_inMaintenance`
+property to `true` and comparing the generated health report with the actual status of
 the services.
 
-A few more tests have been included to verify the basic functionalitiy of the `system` and `inventory`
-services. They can be found under the [hotspot file=2]`src/test/java/it/io/openliberty/guides/inventory/InventoryEndpointTest.java`
-and [hotspot file=3]`src/test/java/it/io/openliberty/guides/system/SystemEndpointTest.java` files. 
-If a test failure occurs, then you might have introduced a bug into the code. These tests 
-will run automatically as a part of the Maven build process when you run the `mvn install` 
-command. You can also run these tests separately from the build by using the `mvn verify` 
-command, but first make sure that the server is stopped.
+A few more tests have been included to verify the basic functionality of the `system` and `inventory`
+services. They can be found under the [hotspot file=2]`src/test/java/it/io/openliberty/guides/inventory/InventoryEndpointIT.java`
```

Comment:
Change
"A few more tests have been included to verify the basic functionality of the `system` and `inventory` services."
To
"A few more tests were included to verify the basic functionality of the `system` and `inventory` services."

Reply:
Updated